### PR TITLE
Add getSummaryData method to return thumbnail and title for summary s…

### PR DESCRIPTION
**Pull request by raissanorth**
…ection of ElementEditor

Required for https://github.com/dnadesign/silverstripe-elemental/pull/284 to allow the required data - content summary - to be pulled in via GraphQL

Requires https://github.com/dnadesign/silverstripe-elemental/pull/313

### CODE REVIEW

File: src/Block/FileBlock.php [hunk 1]
```diff
@@ -39,6 +39,21 @@ public function getSummary()
...
     }
 
+    /**
+     * Return file title and thumbnail for summary section of ElementEditor
+     *
+     * @return array
+     */
+    public function getSummaryData()
+    {
+        $summaryData = [];
```
Comment (ScopeyNZ) [inline src/Block/FileBlock.php]: You should use `$summaryData = parent::getSummaryData()` as a base.

**Review by ScopeyNZ — CHANGES_REQUESTED** [review]

**Comment by robbieaverill** [comment]
Can you please add a `NEW` commit prefix?

**Review by ScopeyNZ — APPROVED** [review]

**Comment by ScopeyNZ** [comment]
Happy now but I'm not sure if the new API is finalised yet on elemental.

**Review by robbieaverill — CHANGES_REQUESTED** [review]
Wait to merge until the elemental parent PR is

**Review by robbieaverill — CHANGES_REQUESTED** [review]
It's kind of unrelated to this PR, but can you please update the composer requirements (and in .travis.yml) to use SS 4.3 and elemental 4.x-dev, so that this code would actually get used in the right context when the automated tests are running

**Review by robbieaverill — APPROVED** [review]

**Pull request merged**